go-controller/pkg/util/ovs.go

```diff
@@ -880,6 +880,31 @@ func DetectCheckPktLengthSupport(bridge string) (bool, error) {
 	return false, nil
 }
 
+// IsOvsHwOffloadEnabled checks if OvS Hardware Offload is enabled.
+func IsOvsHwOffloadEnabled() (bool, error) {
+	stdout, stderr, err := RunOVSVsctl("--no-headings", "--data=bare",
+		"--columns=other_config", "list", "Open_vSwitch", ".")
+	if err != nil {
+		klog.Errorf("Failed to get output from ovs-vsctl --no-headings --columns=other_config "+
+			"--data=bare --columns=other_config list Open_vSwitch . stderr(%s) : %v", stderr, err)
+		return false, err
+	}
+
+	var hwOffloadValue = "false"
+	for _, kvPair := range strings.Fields(stdout) {
+		if strings.HasPrefix(kvPair, "hw-offload=") {
```

Comment:
Do we have the same check_pkt_length problem when tc-policy is set to skip_hw? I wonder if it can return false when:  'hw-offload=true && tc-policy=skip-hw'.

Reply:
I didn't specially check it. But since skip_hw doesn't write the flows into HW, I would say the issue wouldn't be there. However (just by experience debugging the MLX NICs), it is very useful to compare the flows with tc-policy=none and tc-policy=skip_hw. In my opinion, it would be better to keep the flows as similar as possible for debug-ability. I think "skip_hw" is a debugging tool anyways.

Reply:
@zshi-redhat  I hope this is acceptable.